src/app/[locale]/projects/add/page.tsx

```diff
@@ -15,7 +15,7 @@ import { useTranslations } from 'next-intl'
 import { COMMON_NAMESPACE } from '@/object-types/Constants'
 import { useRouter } from 'next/navigation'
 import HttpStatus from '@/object-types/enums/HttpStatus'
-import { TypeOptions, toast } from 'react-toastify'
+import { ToastContainer, TypeOptions, toast } from 'react-toastify'
```

Comment:
Again. I already mentioned that:
Or we assume use along everywhere toastify or we use the ones already available on react-bootstrap.
@hoangnt2 @amritkv We need reach a proper decision.

For the sake of maintenance and consistency, i do not want to have two things doing the same thing on the code.

Reply:
This https://react-bootstrap.github.io/docs/components/toasts or Toastify

Reply:
If it is readily available out of react-bootstrap then we can stick to it instead of external library i.e. react-toastify.
What you think @heliocastro and @hoangnt2 ?

Reply:
@heliocastro @amritkv I agree with this, let's use react-bootstrap's toast